src/workerd/util/sqlite.h

```diff
@@ -522,11 +544,58 @@ class SqliteDatabase::Query final: private ResetListener {
     }
   }
 
+  class QueryEvent {
+   public:
```

Comment:
TIL we half-indent `public/private`, apparently we've done this for 5 months and I literally never noticed :laughing:

Reply:
Heh... yeah.  IIRC, we do that because it makes git output more relevant context lines at the top of diff blocks, like `@@ -522,11 +544,58 @@ class SqliteDatabase::Query [...]` above.